src/plugins/mer/mersettings.cpp

```diff
@@ -209,17 +277,38 @@ void MerSettings::read()
             true).toBool();
     m_qmlLiveBenchLocation = settings->value(QLatin1String(QML_LIVE_BENCH_LOCATION_KEY)).toString();
     m_askBeforeStartingVmEnabled = settings->value(QLatin1String(ASK_BEFORE_STARTING_VM), true).toBool();
-    m_askBeforeClosingVmEnabled = settings->value(QLatin1String(ASK_BEFORE_STARTING_VM), true).toBool();
+    m_askBeforeClosingVmEnabled = settings->value(QLatin1String(ASK_BEFORE_STARTING_VM), true).toBool();    
 
     settings->endGroup();
 
+    settings->beginGroup(QLatin1String(SETTINGS_EMULATOR_CATEGORY));
+    foreach (const QString &name, settings->childKeys()) {
+        const auto map = settings->value(name).value<QVariantMap>();
```

Comment:
I would prefer expanding this as separate keys to make debugging easier

Reply:
Done.